### DISCUSSION TITLE
Screw connections

### DISCUSSION THREAD

**Discussion by ebrahimraeyat** [xcfem/xc#75] [Q&A]
Hi @lcpt. I wanted to know is there any example about analyzing Screw connections in XC, I know it is possible, and you showed me some picture in [this](https://github.com/xcfem/xc/discussions/64#discussioncomment-214450) post.

Thanks.

**Comment by lcpt** [comment]
Hi @ebrahimraeyat.

As I said [here](https://github.com/xcfem/xc/discussions/64#discussioncomment-218520), we need to clean and organize the code before putting some examples in the repository.

**Comment by ebrahimraeyat** [comment]
I want to contribute to it, but I don't know where can I start. is it possible to share it  here. Thanks

**Reply by lcpt** [reply]
I agree that XC is not well documented and the documentation is not well organized, I'm working on it but at a gentle pace (sorry).

The images I have shown in the other discussion belong to a real project and the connections related code is very intermixed with the project itself, so it's not easy to share it (I said any help is welcome, but it was a bit unrealistic because we need to put some order there first). I will prepare small examples for the connections routines and share them soon. For the time being you can take a look to the [gmsh examples](https://github.com/xcfem/xc/tree/master/verif/tests/preprocessor/meshing) that we used to test meshing around boltholes. There area also some examples that use the paving algorithm, but we have discarded this method because it's not very reliable.

Thanks.
